<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_drug_form</class>
 <widget class="QWidget" name="add_drug_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTableWidget" name="drugsTable">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Фильтр</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="filter_text"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="newDrugBtn">
     <property name="text">
      <string>Новое лс</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
